sql/core/src/test/scala/org/apache/spark/sql/execution/QueryExecutionSuite.scala

```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.spark.sql.execution
 
+import java.sql.{Date, Timestamp}
```

Comment:
is it necessary?

Reply:
Yea, let's remove. Looks just a mistake.